app/policies/company_policy.rb

```diff
@@ -1,27 +1,42 @@
 class CompanyPolicy < ApplicationPolicy
-  DEFAULT_ATTRS = %i[name cnpj address phone active code discount].push(user_ids: [])
+  DEFAULT_ATTRS = %i[name cnpj address phone active code].push(user_ids: [])
```

Comment:
Can you remove the `code` field from here? I think it doesn't make sense. Also, check if we have `uniqueness` validation for this field.